test/rest.js

```diff
@@ -875,4 +866,44 @@ describe("REST", function () {
 
   });
 
+  describe("bulk delete", function() {
+    beforeEach(fixture.loadFixtures);
+
+    it("works on individual collections", function(done) {
```

Comment:
I'd put in a test here to make sure there's actually something in the collection.